datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCli.java

```diff
@@ -9,6 +9,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.xml.crypto.Data;
```

Comment:
Is this needed?

Reply:
Nope, no idea why it has been added (automatic imports I guess on bad completion)

Reply:
removed with https://github.com/ICIJ/datashare/pull/1308/commits/52e0305771ce8fb79d5eca2ec5574f221a53f76e